mmv1/products/compute/Interconnect.yaml

```diff
@@ -145,7 +144,6 @@ properties:
       Customer name, to put in the Letter of Authorization as the party authorized to request a
       crossconnect.
     immutable: true
-    required: true
```

Comment:
Could you please update the description of this field to clarify when this field is required and when it is not? Thanks

Reply:
Done.